tests/drivers/entropy/api/Kconfig

```diff
@@ -0,0 +1,10 @@
+# Copyright NXP 2022
+# SPDX-License-Identifier: Apache-2.0
+
+mainmenu "Entropy API Test"
+
+source "Kconfig.zephyr"
+source "tests/drivers/entropy/api/Kconfig.mcux"
```

Comment:
This can be deleted

Reply:
I am confused what you mean, what can be deleted and why?